grains3.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun May  6 20:09:50 2018
+
+@author: Alina
+"""
+
+def get_square_number(benutzereingabe):
+    while True:
+        squares_number = 0
+        try:
+           squares_number = int(benutzereingabe)
+           return squares_number
+        except:
+            benutzereingabe = input("Vorherige Eingabe war keine Zahl! Geben Sie eine Zahl ein:\n")
+            continue
+
+def validate_square_number(squares_number):
+    while True:
+       # valid_squares_number = 0
+       if squares_number < 1 or squares_number > 64:
+           benutzereingabe = input("Zahl nicht zwischen 1 und 64! Wählen Sie eine Zahl zwischen 1 und 64:\n")
+           squares_number = get_square_number(benutzereingabe)
+           continue
+       return squares_number

```

Comment:
Deine beiden Funktionen `get_square_number()` und `validate_square_number()` sind etwas ungeschickt darin, wie sie sich die Arbeit teilen (`input()` steckt in beiden Funktionen, d.h. die Aufgaben sind nicht klar von einander abgegrenzt).